tech-far-hub/gatsby-node.ts

```diff
@@ -142,18 +144,20 @@ export const createPages: GatsbyNode["createPages"] = async ({ graphql, actions,
     }));
     const isIndex = node.pageName === "index";
     let parentPath: string;
-    if (node.pageName === "index") {
-      parentPath = pathParts.join("/");
-    } else {
-      parentPath = pathParts.slice(0, -1).join("/");
-    }
-    const parentPathRegex = `/${parentPath.replace("/", "\\/")}\\/.*/`;
+    parentPath = pathParts.slice(0, -1).join("/");
```

Comment:
I appreciate how you're able to get around these limitations! Seems like one day I want research if there's an easier way. This node file is a keeper tho!